### ISSUE TITLE
2.15 cannot be recognized as a plugin in CC4.4    

### ISSUE THREAD

**Issue by zwl2828596** [soupday/CCiC-Blender-Pipeline-Plugin#56]
2.15 cannot be recognized as a plugin in CC4.4

**Comment by zwl2828596** [comment]
Yes, 2.14 can be put in it, but 2.15 can't. The reason is unclear.

**Comment by soupday** [comment]
Try renaming the folder to something without any 'dots' in the name. Sometimes python doesn't like that.

**Comment by zwl2828596** [comment]
It still doesn't work. I don't know if I'm the only one like this. It seems that I can't use 2.15. Thank you very much for your help.

**Comment by soupday** [comment]
If you extract the plug-in zip somewhere else and run the main.py script from the **Script** > **Load Python** menu. It will run the plug-in temporarily as if it has been installed.

If anything goes wrong with that it will show the error where it went wrong in the **Script** > **Console Log**, and that error, if it happens, should reveal the problem.

**Comment by indigo1371** [comment]
Greeting) I have same problem with plugin and CC4.4, but here logo:
<class 'NameError'>,  File: C:/Program Files/Reallusion/Character Creator 4/Bin64/OpenPlugin\btp\vars.py,  Line: 28,  Function: <module>

**Comment by soupday** [comment]
> Greetings) I have same problem with plugin and CC4.4, but here log: <class 'NameError'>, File: C:/Program Files/Reallusion/Character Creator 4/Bin64/OpenPlugin\btp\vars.py, Line: 28, Function:

You missed out the parent folder it should be .../Bin64/OpenPlugin/CCiC-Blender-Pipeline-Plugin-main/...

**Comment by soupday** [comment]
Update character creator. I think you're a version behind.

**Comment by diemvan25789** [comment]
Hi, I have same problem with plugin and CC4.4
> Greetings) I have same problem with plugin and CC4.4, 
log: <class 'NameError'>,  File: C:/Program Files/Reallusion/iClone 8/Bin64/OpenPlugin/2\btp\vars.py,  Line: 28,  Function: <module>
<class 'AttributeError'>,  File: unknown,  Line: unknown,  Function: unknown
Please help me fix thank pro soupday
<img width="708" alt="image" src="https://github.com/user-attachments/assets/3e4d5997-b9f2-46d3-b544-21fa57d1b8da">
<img width="573" alt="image" src="https://github.com/user-attachments/assets/b15486b1-ce3b-4af9-ba1c-2c1bcda0067b">
<img width="168" alt="image" src="https://github.com/user-attachments/assets/eb65dec3-eca6-4b67-b0a3-cbe9f62fac8e">

**Comment by soupday** [comment]
Current version of iClone is 8.42.3014.1

Can you upgrade to this or is it not available to you yet?

**Comment by diemvan25789** [comment]
> Current version of iClone is 8.42.3014.1
> 
> Can you upgrade to this or is it not available to you yet?

Thank you, love you, keep it up bro!
